doc/debug-messages.txt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+
```

Comment:
> Since this is apparently a list devs are going to have to maintain manually

It is not. See item 1 in my introduction to https://github.com/squid-cache/squid/pull/775#pullrequestreview-590722896

Reply:
Okay. I would still like these details to be included in the web documentation. So a title would help even with auto-generation.

Reply:
> Double empty line.

Removed in commit 301be22. Please note that this introduces a yet another inconsistency with debug-sections.txt.

> please add column titles to clarify what the values are

Added in commit fc4659a.